app/_how-tos/add-rate-limiting-tiers-with-kong-gateway.md

```diff
@@ -76,7 +78,11 @@ entities:
 {% endcapture %}
 {{ plugin | indent: 3 }}
 
-1. Create the Free, Basic, and Premium tier consumer groups:
+### 2. Create consumer groups for each tier
+
+Before you can enable rate limiting for tiers of users, we first have to create consumer groups for each tier and then add consumers to those groups. Consumer groups are soley a way to organize consumers of your APIs. In this guide, we'll create three tiers (Free, Basic, and Premium), so we need to create a unique consumer group for each tier.
+
+Apphend the following content to your `kong.yaml` file in the `deck_files` directory to create consumer groups for each tier:
 
 {% capture groups %}
```

Comment:
Since these are no longer indented into ordered lists, you can remove the `capture` tags and `indent` filters, and just leave the `entity_examples` blocks.